pkg/operator/controllers/monitoring/monitoring_controller.go

```diff
@@ -50,23 +50,25 @@ type Config struct {
 	} `json:"alertmanagerMain,omitempty"`
 }
 
-type Reconciler struct {
+type MonitoringReconciler struct {
+	log *logrus.Entry
+
 	arocli        aroclient.Interface
 	kubernetescli kubernetes.Interface
-	log           *logrus.Entry
-	jsonHandle    *codec.JsonHandle
+
+	jsonHandle *codec.JsonHandle
 }
 
-func NewReconciler(log *logrus.Entry, kubernetescli kubernetes.Interface, arocli aroclient.Interface) *Reconciler {
-	return &Reconciler{
+func NewReconciler(log *logrus.Entry, arocli aroclient.Interface, kubernetescli kubernetes.Interface) *MonitoringReconciler {
```

Comment:
If `Reconciler` -> `MonitoringReconciler` ranaming is going to actually happen, then we also need to rename `NewReconciler` to `NewMonitoringReconciler`

Reply:
I'll follow your suggestion to choose more generic type names, and will thus leave this function name unchanged.